suite-native/module-dev-utils/src/screens/DemoScreen.tsx

```diff
@@ -301,13 +300,6 @@ export const DemoScreen = () => {
                             />
                         </Box>
                     </Box>
-
-                    <NumPadButton
```

Comment:
Would be nice to have button here anyway.

Reply:
[a376ae7](https://github.com/trezor/trezor-suite/pull/9479/commits/a376ae77aaf2859f0631845b07abca2da22cac44)